whisperstream/stream.py

```diff
@@ -165,8 +175,10 @@ async def atranscribe_streaming(
         kwargs["language"] = language.pt1
 
     if force_punctuation:
-        if kwargs.get("prompt") is None:
-            raise ValueError("cannot set both prompt and force_punctuation")  # we do not know what to do TODO
+        if kwargs.get("prompt") is not None:  # checking if user added initial prompt
```

Comment:
this comment does not add any sense. Instead add comment which explains that it is unclear what to do in case user provided custom prompt